### ISSUE TITLE
Datepicker keyboard navigation on Chrome doesn't work

### ISSUE THREAD

**Issue by riy** [angular-ui/bootstrap#5754]

**Comment by deeg** [comment]
This seems to be working for me.

I am testing on:
Mac OSX 10.10.5
Chrome Version 49.0.2623.110 (64-bit)

**Comment by riy** [comment]
Thanks for testing this, Dan. I just tested this again, on another (older) MacBook Pro - same problem there. It's definitely not the setup on my machine. You and I are both testing this on the same Chrome version, only the Mac OSX version differs. But then again, I had the same issue on Windows/Chrome.

Did you use the [link](http://angular-ui.github.io/bootstrap/versioned-docs/1.3.1/) mentioned above?

**Comment by deeg** [comment]
@riy, yes using the link you provided, I went down to either the datepicker or datepicker popup section and opened the calendar. Then I started to use the arrow keys to switch to different dates, and enter to select that date.

Are you seeing any errors in the console?

**Comment by riy** [comment]
There are no errors at all in the JavaScript console. I also tried this in Chrome's incognito mode, hoping that maybe it's just some add-on interfering with the JavaScript, but that also didn't work. Also my colleagues with their Macs and Windows machines are having the same issue, not just me.

I have also tested this locally, as I thought maybe the company proxy server/firewall is messing with the JavaScript, but same issue here.

**Comment by riy** [comment]
Hey, I just tested the datepicker-popup - that one works fine!
Then I went back to the datepicker and switched to the next month, but didn't select a date, just started using the keyboard - that also worked fine.

So what happens is when a date is selected using the mouse (meaning: when you click on a date) then the keyboard navigation stops working.

**Comment by deeg** [comment]
Even after clicking a date and selecting it with the mouse, I can then start using the arrow keys to start choosing a new date and move around the calendar.

**Comment by riy** [comment]
I managed to find a solution for our particular problem (using keyboard navigation in automated Protractor testing for choosing a date x days in the future) but the problem itself remains.

Maybe someone else reading this can test this as well?

**Comment by icfantv** [comment]
I can confirm this is an issue on our demo page, but only in Chrome.  I did not test with all my extensions disabled.  Just use the displayed datepicker, don't use the popup to see the issue.

It works fine in Safari and Firefox so I'm going to suggest this issue be closed as a browser issue and there's most likely nothing we can do about it.

**Comment by wesleycho** [comment]
This looks like an issue with focus management in Chrome. We need to look into this, but I think this is a bug.
